### PR TITLE
ST-1388 ST CIC bundle config file

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ST-1388

### Change description ###
ST CIC bundle config file

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+hasCoversheets: true
+hasFolderCoversheets: true
+folders:
+  - name: Application Documents
+    documents:
+      - type: documentSet
+        property: /caseDocuments
+        filters:
+          - property: /documentType
+              value: A
```
Comment (mdayican) [inline src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml:14]: wrong indentation

File: src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  - name: Further additions
+    documents:
+      - type: documentSet
+        property: /caseDocuments
+        filters:
+          - property: /documentType
+              value: TD
```
Comment (mdayican) [inline src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml:21]: you can not add multiple "value" like this you need to use "-" and as I remember this property does not support array.

Comment (yogesh-hullatti) [reply src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml:21]: example : https://github.com/hmcts/em-ccd-orchestrator/blob/master/src/main/resources/bundleconfiguration/sscs-bundle-config.yaml#L12

https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1201996412#AutomatedBundlingConfigurationFileGuide-FilteringDocuments

[the remark this answers is not in the document]
Comment (ElifDenizEsea) [reply src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml]: Could you please check again?

File: src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+          - property: /documentType
+            value: A
+  - name: Further additions
+    documents:
+      - type: documentSet
+        property: /caseDocuments
+        filters:
+          - property: /documentType
+              value:
+                - TD
```
Comment (mdayican) [inline src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml:22]: no arrays, single value. 
https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1201996412#AutomatedBundlingConfigurationFileGuide-FilteringDocuments

File: src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        filters:
+          - property: /documentType
+              value: L
+  - name: General evidence
+    documents:
+      - type: documentSet
+        property: /caseDocuments
+        filters:
+          - property: /documentType
+              value: TG
```
Comment (mdayican) [inline src/main/resources/bundleconfiguration/st_cic_bundle_all_case.yaml:63]: all "value" properties has wrong indentation

**Review by mdayican — APPROVED** [review]

**Pull request merged**